packages/circuit-ui/components/PhoneNumberInput/PhoneNumberInput.tsx

```diff
@@ -382,7 +397,18 @@ export const PhoneNumberInput = forwardRef<
                 countryCodeRef as RefObject<HTMLInputElement>,
                 countryCode.ref as ForwardedRef<HTMLInputElement>,
               )}
-              renderPrefix={countryCode.renderPrefix}
+              renderPrefix={
+                (countryCode.renderPrefix as InputProps['renderPrefix']) ??
+                (({ value: inputValue, ...rest }) => (
+                  <DefaultPrefix
+                    value={getCountry(
+                      countryCode.options,
+                      inputValue?.toString(),
```

Comment:
We know for sure that the input value is a string here, even if TypeScript isn't sure.

```suggestion
                      inputValue as string,
```